crates/turborepo-lib/src/run/scope/change_detector.rs

```diff
@@ -70,23 +69,21 @@ impl<'a> ScopeChangeDetector<'a> {
             &lockfile_path,
         ) {
             debug!("lockfile did not change");
-            return None;
```

Comment:
this appears to have been a bug.  the "None" result was intended to mean "there was a change but we don't know what it was" which maps, today, to `LockfileContents::Unknown`.  But, as you can see here, it's being used in the codepath where `LockfileContents::Unchanged` is correct.

Reply:
update: it was both but they hit the same codepath - so there's no behavioral change here here.